src/stores/project-store.js

```diff
@@ -0,0 +1,42 @@
+const STORAGE_KEY = 'leto-projects';
+
+export const projectStore = {
```

Comment:
is not a store structure, please follow https://pinia.vuejs.org/introduction.html#basic-example
to make a real store.

But I'm not sure that we want a store here, may be add all these functions in the Project composable